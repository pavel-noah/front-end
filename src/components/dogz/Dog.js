import React from 'react';
import PropTypes from 'prop-types';

export default function Dog({ dogName, dogAge, dogWeight }) {
    return (<>
            <h1>{dogName}</h1>
            <ul style={{ listStyle: 'none' }}>
                <li>Age: {dogAge} </li>
                <li>Weight: {dogWeight} </li>
            </ul>
        </>
    );
}

Dog.propTypes = {
    dogName: PropTypes.string.isRequired,
    dogAge: PropTypes.string.isRequired,
    dogWeight: PropTypes.string.isRequired
};
