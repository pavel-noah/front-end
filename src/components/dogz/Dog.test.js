import React from 'react';
import { shallow } from 'enzyme';
import Dog from './Dog';

describe('dog component', () => {
    it('renders dog', () => {
        const wrapper = shallow(<Dog dogName="" dogAge="" dogWeight="" />);
        expect(wrapper).toMatchSnapshot();
    });
});
  
