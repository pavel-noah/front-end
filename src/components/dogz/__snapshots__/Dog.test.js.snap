// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dog component renders dog 1`] = `
<Fragment>
  <h1 />
  <ul
    style={
      Object {
        "listStyle": "none",
      }
    }
  >
    <li>
      Age: 
       
    </li>
    <li>
      Weight: 
       
    </li>
  </ul>
</Fragment>
`;
